cohorts/plot.py

```diff
@@ -22,7 +22,38 @@
 from sklearn.metrics import roc_curve
 from .model import bootstrap_auc
 
-def stripboxplot(x, y, data, ax=None, **kwargs):
+def vertical_percent(plot, percent=0.1):
+    plot_bottom, plot_top = plot.get_ylim()
+    return percent * (plot_top - plot_bottom)
+
+def set_min_y_from_data(y, data, plot):
+    """
+    If data is >= 0, don't show negative axis values.
+    """
+    min_y_val = data[y].min()
+    if min_y_val >= 0:
+        # Add a bit of padding so that we don't cut the plot off in an ugly way
+        extra_space = -1 * vertical_percent(plot, 0.05)
+        plot.set_ylim(bottom=extra_space)
```

Comment:
Interesting; maybe that's a more stable approach? Not sure the best way to automatically set the max; hard to know if we're talking about 0 to 100 or not?